asv_bench/benchmarks/groupby.py

```diff
@@ -132,3 +132,34 @@ def setup(self, *args, **kwargs):
         super().setup(**kwargs)
         self.ds1d = self.ds1d.chunk({"time": 50})
         self.ds2d = self.ds2d.chunk({"time": 50, "z": 4})
+
+
+class ResampleCFTime:
```

Comment:
I think you are missing some class inheritance or otherwise there won't be any benchmark?